### DISCUSSION TITLE
TLS version

### DISCUSSION THREAD

**Discussion by ghost** [tomer8007/kik-bot-api-unofficial#183] [General]
Hey, I know it is off-topic but can you tell me what TLS version Kik messenger has implementet? I want to learn something about its security since there is no proper security documentation. I hope it uses TLS 1.3 otherwhise it would be quite insecure due to attacking capabilities...
Thanks.

**Comment by tomer8007** [comment]
Hi, thanks for using the Discussions page.
 I don't remember exactly which TLS version the original Android application uses. I can at least tell you that their servers accept TLS 1.2 because this is what this API uses.
Anyway if you want to know which version the app uses, that would be really easy, just fire up Wireshark. and look for sessions with `talk1110an.kik.com` or similar. You don't even have to decrypt or hook anything.

**Reply by tomer8007** [reply]
I would say that what you said about E2E is correct.
